enderio-machines/src/main/java/crazypants/enderio/machines/machine/teleport/anchor/BlockTravelAnchor.java

```diff
@@ -160,6 +160,22 @@ public void onBlockPlacedBy(@Nonnull World world, @Nonnull BlockPos pos, @Nonnul
     }
   }
 
+  @Override
+  public boolean removedByPlayer(@Nonnull IBlockState state, @Nonnull World world, @Nonnull BlockPos pos, @Nonnull EntityPlayer entityPlayer,
+      boolean willHarvest) {
+    TileEntity te = world.getTileEntity(pos);
+
+    ITravelAccessable ta = (ITravelAccessable) te;
+    if (ta.getOwner().equals(UserIdent.create(entityPlayer.getGameProfile())) || (ta.getAccessMode() == ITravelAccessable.AccessMode.PUBLIC)) {
+      return super.removedByPlayer(state, world, pos, entityPlayer, willHarvest);
+    } else {
+
+      ChatUtil.sendNoSpam(entityPlayer, Lang.GUI_HARVEST_ERROR_PRIVATE.toChatServer(TextFormatting.RED + ta.getOwner().getPlayerName() + TextFormatting.WHITE));
```

Comment:
Please use player.sendStatusMessage(..., true). ChatUtil.sendNoSpam() should no longer be used for one-line messages. (changing the existing ones is a Trello task)

Please put the colors into the lang file. §4 and §f, I think